CorsixTH/Lua/epidemic.lua

```diff
@@ -112,6 +112,7 @@ function Epidemic:tick()
       self:checkNoInfectedPlayerHasLeft()
       self:markedPatientsCallForVaccination()
       self:showAppropriateAdviceMessages()
+      self:checkThereAreNoAnyInfectedPatientsLeft()
```

Comment:
Function name got a little garbed here, where's it referencing?

Reply:
> Function name got a little garbed here, where's it referencing?

Oh, true. thanks.